cdec/installation-manager-core/src/main/java/com/codenvy/im/managers/BackupConfig.java

```diff
@@ -69,7 +70,7 @@ public Path getRelativeBackupPath() {
     }
 
     protected static String BACKUP_NAME_TIME_FORMAT = "dd-MMM-yyyy_HH-mm-ss";
-    public static    Path   BASE_TMP_DIRECTORY      = Paths.get(System.getProperty("java.io.tmpdir")).resolve("codenvy");
+    public static    Path   BASE_TMP_DIRECTORY      = Paths.get("/tmp").resolve("codenvy");
```

Comment:
Because of **postgres** account, which is used to dump/restore database, doesn't have an access to the _/home/vagrant/codenvy/cli/data/tmp_ directory.